### DISCUSSION TITLE
How can I make rigid_object not collide with others.

### DISCUSSION THREAD

**Discussion by WSYANGSX** [isaac-sim/IsaacLab#597] [Q&A]
Hello! 
When I use IsaacLab to build my own env, I found two rigid_objects can still collide with each other although I have set collision_props to False. I want to know how to make them not collide with each other.Thank you!
![image](https://github.com/isaac-sim/IsaacLab/assets/99880178/6f540a4f-05f5-4b9a-b526-97b22b9fcc34)
And I find some Warning on termination.
![image](https://github.com/isaac-sim/IsaacLab/assets/99880178/e5efd25b-99ab-44b5-8591-2453156cf5c4)

**Comment by ahanjaya** [comment]
Same here under the same environment, it will collide with each other. I tried ` sim_utils.CollisionPropertiesCfg(collision_enabled=False)`, got the same warning and object still collide with each other
